### PR TITLE
feat: updated unauthenticated views on both publish and responses form builder tabs

### PR DESCRIPTION
**Pull request by craigzour**
# Summary | Résumé

closes https://github.com/cds-snc/platform-forms-client/issues/1869

- Updated unauthenticated views on both publish and responses form builder tabs.
- Enabled form title in responses view (instead of static GC Forms). Got the approval from @samsadasivan 

| Before                                          | After                                        |
| ----------------------------------------------- | -------------------------------------------- |
|Screenshot 2023-05-12 at 9 15 52 AM|Screenshot 2023-05-12 at 9 16 47 AM|
|Screenshot 2023-05-12 at 9 15 58 AM|Screenshot 2023-05-12 at 9 16 51 AM|
|Screenshot 2023-05-12 at 9 16 24 AM|Screenshot 2023-05-12 at 9 16 40 AM|

# Test instructions | Instructions pour tester la modification

- Make sure you are not authenticated
- Visit `https://forms-staging.cdssandbox.xyz/form-builder/publish`
- Visit `https://forms-staging.cdssandbox.xyz/form-builder/responses`

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [ ] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [ ] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [ ] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [ ] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [ ] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [ ] Have you modified the change log and updated any relevant documentation?
- [ ] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [ ] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

File: pages/form-builder/responses/[[...params]].tsx [hunk 1]
```diff
@@ -55,12 +55,19 @@ const Responses: NextPageWithLayout<ResponsesProps> = ({
...
+          <LoggedOutTab tabName={LoggedOutTabName.RESPONSES} />
+        </PageTemplate>
+      </>
+    );
+  }
+
   if (deliveryOption && deliveryOption.emailAddress) {
     return (
       <>
-        <Head>
```
Comment (craigzour) [inline pages/form-builder/responses/[[...params]].tsx:61]: Removed `Head` as it is already included in the `PageTemplate` component we use below.

File: public/static/locales/fr/form-builder.json [hunk 2]
```diff
@@ -493,6 +481,25 @@
...
     }
   },
+  "tabLoggedOut": {
+    "publish": {
+      "title": "Vous ne pouvez pas modifier les paramètres",
+      "text1": "Pour modifier les paramètres de ce formulaire,"
+    },
+    "responses": {
+      "title": "Vous ne pouvez pas modifier les paramètres",
+      "text1": "Pour modifier les paramètres de ce formulaire,"
```
Comment (anikbrazeau) [inline public/static/locales/fr/form-builder.json:491]:
```suggestion
      "text1": "Pour accéder aux réponses du formulaire,"
```

File: public/static/locales/en/form-builder.json [hunk 3]
```diff
@@ -492,6 +480,25 @@
...
+    },
+    "responses": {
+      "title": "You cannot access responses",
+      "text1": "To access responses of this form,"
+    },
+    "settings": {
+      "title": "You cannot change settings",
+      "text1": "To change the settings of this form,"
+    },
+    "text2": "sign into your account",
```
Comment (anikbrazeau) [inline public/static/locales/en/form-builder.json:496]:
```suggestion
    "text2": "sign in to your account",
```

File: public/static/locales/fr/form-builder.json [hunk 4]
```diff
@@ -493,6 +481,25 @@
       }
     }
   },
+  "tabLoggedOut": {
+    "publish": {
+      "title": "Vous ne pouvez pas modifier les paramètres",
```
Comment (anikbrazeau) [inline public/static/locales/fr/form-builder.json:486]:
```suggestion
      "title": "Vous ne pouvez pas publier",
```

Comment (anikbrazeau) [inline public/static/locales/fr/form-builder.json:486]:
```suggestion
      "title": "Vous ne pouvez pas publier",
```

File: public/static/locales/fr/form-builder.json [hunk 5]
```diff
@@ -493,6 +481,25 @@
...
+      "text1": "Pour modifier les paramètres de ce formulaire,"
```
Comment (anikbrazeau) [inline public/static/locales/fr/form-builder.json:487]:
```suggestion
      "text1": "Pour publier ce formulaire,"
```

**Review by thiessenp-cds — APPROVED** [review]
LGTM :)

Nice work! I cross browser tested and it looks/sounds (screen readers :) good.

Future Thought: Having a heading like H2 or H3 or configurable heading would help add structure to pages. This is more a reflection on myself since I wrote the original card component.. :)

**Pull request merged**